api/src/graphql/resolvers/index.js

```diff
@@ -120,6 +122,8 @@ const resolvers = {
     },
   },
   Mutation: {
+    initLoomioDiscussions: () => initLoomioDiscussions(),
+    initLoomioGroups: () => initLoomioGroups(),
```

Comment:
Interesting. Do we need to add this to the GraphQL API? Why does the frontend need to be able to access this? Or is it preparing for a future where our systems will be talking to each other through APIs?

Reply:
Removed.